docs/design/two-phase-transfers.md

```diff
@@ -24,28 +24,24 @@ increased on the relevant accounts at this point.
 * [`post_pending_transfer`](../reference/transfers.md#flagspost_pending_transfer): Move all (or part) of the reserved funds to the pending transfer's destination.
 * [`void_pending_transfer`](../reference/transfers.md#flagsvoid_pending_transfer): Revert all of the reserved funds to the original account.
 
-A pending transfer can only be posted or voided once. It cannot be
-posted twice or voided then posted, etc.
+## Resolving a two-phase transfer
 
 When the pending transfer is resolved (posted or voided), the
 [`debits_pending`](../reference/accounts.md#debits_pending) and
 [`credits_pending`](../reference/accounts.md#credits_pending) fields
 on the respective accounts are decreased by the
 [`amount`](../reference/transfers.md#amount) of the **pending** transfer.
 
+### Posting
```

Comment:
Nit: I would change these headers from "Posting"/"Voiding"/"Timeouts" to "Post"/"Void"/"Timeout" to make them more active verbs.

Reply:
Done